C7GameData/RulesFormat.cs

```diff
@@ -43,6 +43,9 @@ public class C7RulesFormat
 		public StrengthBonus cityLevel2DefenseBonus;
 		public StrengthBonus cityLevel3DefenseBonus;
 
+		public int healRateInField;
+		public int healRateInCity;
```

Comment:
really, there should be options for friendly, neutral, and hostile field, possibly in transport, and I suppose any other conditions (eg fortress) would be modifiers